Python/pywarpx/picmi.py

```diff
@@ -545,6 +545,12 @@ def initialize_inputs(self):
             pywarpx.warpx.do_electrostatic = 'labframe'
             pywarpx.warpx.self_fields_required_precision = self.required_precision
             pywarpx.warpx.self_fields_max_iters = self.maximum_iterations
+            pywarpx.boundary.potential_lo = [
+                self.grid.potential_xmin, self.grid.potential_ymin
```

Comment:
This would need the corresponding change in the picmi standard to add the potential_xmin etc input parameters. For now, the thing to do would be to allow the parameters `warpx_potential_xmin` etc in the Grid classes here and set the `potential_xmin` etc attributes in the init methods for those classes.

Reply:
I had forgotten that I locally edited the picmi standard to add potential_xmin, etc. I've changed all three Grid classes (2d, 3d and CylindricalGrid) to pop `warpx_potential_xmin`, etc. from the inputs. Note that for `CylindricalGrid` I force the x and z potentials to be 0 and set `self.potential_ymin = kw.pop('warpx_potential_zmin', 0)` since the z-direction is the only one that has Dirichlet boundary conditions implemented, but the value is expected in the second entry for `boundary.potential_lo` and `boundary.potential_hi`.